packages/app/src/components/FaucetApp.tsx

```diff
@@ -46,6 +46,7 @@ export function FaucetApp({ onSuccess }: FaucetAppProps) {
         className="mt-5"
         isLoading={faucet.mutation.isLoading}
         onPress={() => faucet.handleFaucet(captcha.value)}
+        data-testid="addFundsBtn"
```

Comment:
https://testing-library.com/docs/queries/bytestid

> Using data-testid attributes do not resemble how your software is used and should be avoided if possible

I think @pedronauck had add `aria-` on the other fields it improves the accessibility and helps on tests

Reply:
what aria has to do with data-testid? I dont understand what exactly you want with this comment

you want me to add aria prop as well to improve accessibility ?

Reply:
this section has a good explanation about this
https://testing-library.com/docs/queries/byrole/